File: fixtures/node_output/post_with_data_raw.js
var request = require('request');

var dataString = 'msg1=wow&msg2=such&msg3=@rawmsg';

var options = {
    url: 'http://example.com/post',
    method: 'POST',
    body: dataString
};

function callback(error, response, body) {
    if (!error && response.statusCode == 200) {
        console.log(body);
    }
}

request(options, callback);

File: fixtures/elixir_output/post_with_data_raw.ex
request = %HTTPoison.Request{
  method: :post,
  url: "http://example.com/post",
  options: [],
  headers: [],
  params: [],
  body: [
    {~s|msg1|, ~s|wow|},
    {~s|msg2|, ~s|such|},
    {~s|msg3|, ~s|@rawmsg|}
  ]
}

response = HTTPoison.request(request)
